packages/@n8n/task-runner-python/src/logs.py

```diff
@@ -70,3 +76,23 @@ def setup_logging():
     logger.addHandler(stream_handler)
 
     logging.getLogger("websockets.client").setLevel(logging.DEBUG)
+
+
+def format_print_args(*args) -> list[str]:
+    """Format `print` arguments for browser console output, handling circular references."""
```

Comment:
A bit difficult to follow what happens here as there's no info what `*args` really are.

A test case or some examples would make this clearer